common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -328,6 +328,8 @@ def get_structure(self, key, course_context=None):
 
                 with TIMER.timer("get_structure.find_one", course_context) as tagger_find_one:
                     doc = self.structures.find_one({'_id': key})
+                    if doc is None:
```

Comment:
It looks like if head validation doesn't happen (split.py L865), it's perfectly possible to pass in a course key that is valid but doesn't exist, and then get a None back from that call.